modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_content-type: PROCEDURE
+[id="ztp-configuring-pgt-compliance-eval-timeouts_{context}"]
+= Configuring compliance evaluation timeouts for PolicyGenTemplate CRs
+
+You can configure PolicyGenTemplate CRs with duration settings for how long a cluster can be in a state of policy compliance or non-compliance before {rh-rhacm} re-evaluates the applied cluster policies. The default value for the `noncompliant` state is 10 seconds. The default value for the `compliant` state is 10 minutes. To disable the evaluation interval, set the value to `never`.
```

Comment:
The duration settings are applied to a ConfigurationPolicy and they are not cluster wide configuration.

Reply:
Updated. PTAL